install/kubernetes/cilium/values.yaml

```diff
@@ -263,6 +263,11 @@ global:
     # mode is the mode of NodePort feature
     mode: "hybrid"
 
+  # hostPort is the configuration for container hostPort mapping
+  hostPort:
+    # enabled enables the hostPort functionality
+    enabled: false
```

Comment:
Do we want to disable the hostPort feature for new users? Asking, as for the existing users after the upgrade to v1.8 the feature will be enabled due to the agent defaulting the flag to `true`.

Reply:
I would keep it enabled otherwise people might miss it and still deploy the `portmap` CNI chaining. I'll do a follow-up for the upgrade guide to document the support and actions needed.